server/src/main/kotlin/core/gameflow/handstate/HandStateExtensions.kt

```diff
@@ -7,8 +7,7 @@ import core.gameflow.player.Player
 import core.gameflow.Positions
 
 fun HandState.updateActivePlayer(playerUpdate: Player): HandState {
-
-    assert(activePlayer != null)
+    activePlayer!!
```

Comment:
how is it different than the deleted line? just asking xd

Reply:
It is not, it just looks a bit nicer imo